### PR TITLE
Add Dance : Tap

### PR DESCRIPTION
**Pull request by lim4349**
Use matrix function, and it looks like someone tap dancing!

### CODE REVIEW

**Comment by Okazari** [comment]
tilt
Thanks for your contribution, gonna look at the code :)

**Comment by Okazari** [comment]
Looking at the animation, i would suggest to rename it *Tilt*
What do you think ?

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (Okazari) [inline .idea/codeStyles/Project.xml]: This file should not be commited

File: .idea/dbnavigator.xml [hunk 2]
```diff
@@ -0,0 +1,453 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Okazari) [inline .idea/dbnavigator.xml]: This file should not be commited

File: .idea/modules.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Okazari) [inline .idea/modules.xml]: This file should not be commited

File: .idea/vcs.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Okazari) [inline .idea/vcs.xml]: This file should not be commited

File: .idea/workspace.xml [hunk 5]
```diff
@@ -0,0 +1,766 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Okazari) [inline .idea/workspace.xml]: This file should not be commited

File: src/dances/tap.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+export default (elem, value, options = {}) => {
+  const options =
```
Comment (Okazari) [inline src/dances/tap.js]: You don't need to do this here. `options = {}` already intialize `options` to an empty object if undefined.
You also have a name conflict

File: src/dances/tap.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    arguments.length > 2 && arguments[2] !== undefined ? arguments[2] : {}
+
+  const max = !isNaN(options.max) ? options.max : 25
+  const min = !isNaN(options.min) ? options.min : 20
+  const rotate3d = (max - min) * value
```
Comment (Okazari) [inline src/dances/tap.js]: you must use `let` here instead of `const`, since you reafect the variable.

File: src/dances/tap.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+  if (options.reverse) {
+    rotate3d = max - rotate3d
+  } else {
```
Comment (Okazari) [inline src/dances/tap.js]: This `else` is not usefull in this case, you can remove the lines 10 / 11 / 12

**Comment by lim4349** [comment]
i fixed what you say, and i pull-request again! check it please.

**Pull request closed without merging**